content/01.proposal.md

```diff
@@ -3,10 +3,15 @@
 Manubot is a system for writing scholarly documents on GitHub.
 Manubot aims to **transform publishing** to be transparent & reproducible, immediate & permissionless, versioned & automated, collaborative & open, linked & provenanced, decentralized & hackable, interactive & annotated, and free of charge.
 
-Manubot applies **tactics from open source** software to manuscript writing.
+Manubot transfers the lessons from the **open source software movement** to academic writing.
+Documents are written in the open, creating an easy path for readers to become contributors.
+We recommend following a workflow where anyone can propose changes, which are then reviewed by project members and revised as necessary before being incorporated.
```

Comment:
Cite meta-review here?

Reply:
We link to the Meta Review below. I was actually thinking we'd want to add something like:

> (for example, [see how](https://github.com/greenelab/manufund-2018/pull/2) this proposal was created)

Which links to this PR. What do you think?

Reply:
Clever! That sounds like a good idea.